src/commands.ts

```diff
@@ -331,85 +325,44 @@ async function addCMD(shell: Shell) {
     return connectCMD(shell, [gate.name])
 }
 
-async function resetCMD(shell: Shell, args: string[]) {
-    let gate: Gate
-    if (args[0]) {
-        gate = shell.getGate(args[0])
-        if (!gate)
-            return shell.t.writeln(`Host not found: ${args[0]}`)
-    } else {
-        gate = terminal7.activeG
-        if (!gate)
-            return shell.t.writeln("No active connection")
-    }
-    const fields = [
-        { prompt: "Close gate" },
-        { prompt: "Reset connection & Layout" },
-        { prompt: "\x1B[31mFactory reset\x1B[0m" },
+async function resetCMD(shell: Shell) {
```

Comment:
I guess I wasn't clear enough. I like what the reset command does when I'm connected to a gate and I don't want to change it. I just want to add this reset when the users keys a reset with no parameters and there's no active gate.